src/main/java/com/socialmedia/controller/UserController.java

```diff
@@ -38,10 +39,11 @@ public UserController(AuthenticationService authenticationService, UserMapper us
   }
 
   @PostMapping
-  public ResponseEntity<Token> signUp(@RequestBody UserCredentials credentials) {
+  public ResponseEntity<Token> signUp(@RequestBody UserRegistrationDtoIn userForm) {
 
-    userMapper.createFromCredentials(credentials);
-    Token token = new Token(authenticationService.getAccessToken(credentials));
+    userMapper.create(userForm);
+    UserCredentials credentials = new UserCredentials(userForm.getUsername(), userForm.getPassword());
```

Comment:
to be consistent move all this stuff to mapper layer

Reply:
done